conf.py

```diff
@@ -111,61 +111,61 @@
 html_sidebars = {
     # Defaults taken from http://www.sphinx-doc.org/en/stable/config.html#confval-html_sidebars
     # Removes the quick search block
-    '**': ['localtoc.html', 'globaltoc.html', 'relations.html', 'customsourcelink.html'],
+    '**': ['localtoc.html', 'globaltoc.html', 'relations.html', 'customsourcelink.html']
```

Comment:
Trailing comma makes the dict easier for modification.

Reply:
and hence is considered good practice.